File: src/test/java/no/nav/syfo/testhelper/UserConstants.java
package no.nav.syfo.testhelper;

import static no.nav.syfo.testhelper.MockAktorRegKt.mockAktorId;

public class UserConstants {

    public static final String ARBEIDSTAKER_FNR = "12345678912";
    public static final String ARBEIDSTAKER_AKTORID = mockAktorId(ARBEIDSTAKER_FNR);
    public static final String LEDER_FNR = "12987654321";
    public static final String LEDER_AKTORID = mockAktorId(LEDER_FNR);
    public static final String VIRKSOMHETSNUMMER = "1234";
    public static final String NAV_ENHET = "0330";
    public static final String NAV_ENHET_NAVN = "Bjerke";
    public static final String VEILEDER_ID = "Z999999";
    public static final String VEILEDER_2_ID = "Z888888";
    public static final String STS_TOKEN = "123456789";

    public static final String PERSON_NAME_FIRST = "First";
    public static final String PERSON_NAME_MIDDLE = "Middle";
    public static final String PERSON_NAME_LAST = "Last";

    public static final String PERSON_FULL_NAME = PERSON_NAME_FIRST + " " +  PERSON_NAME_MIDDLE + " " + PERSON_NAME_LAST;
}

File: src/test/kotlin/no/nav/syfo/testhelper/UserConstants.kt
package no.nav.syfo.testhelper

object UserConstants {
    const val ARBEIDSTAKER_FNR = "12345678912"
    const val ARBEIDSTAKER_FNR_2 = "5475678915"
    private const val MOCK_AKTORID_PREFIX = "10"

    @JvmField
    val ARBEIDSTAKER_AKTORID = mockAktorId(ARBEIDSTAKER_FNR)
    const val LEDER_FNR = "12987654321"

    @JvmField
    val LEDER_AKTORID = mockAktorId(LEDER_FNR)
    const val VIRKSOMHETSNUMMER = "123456789"
    const val VIRKSOMHETSNUMMER_2 = "123456780"
    const val NAV_ENHET = "0330"
    const val NAV_ENHET_NAVN = "Bjerke"
    const val VEILEDER_ID = "Z999999"
    const val VEILEDER_2_ID = "Z888888"
    const val STS_TOKEN = "123456789"
    const val PERSON_NAME_FIRST = "First"
    const val PERSON_NAME_MIDDLE = "Middle"
    const val PERSON_NAME_LAST = "Last"
    const val PERSON_FULL_NAME = "$PERSON_NAME_FIRST $PERSON_NAME_MIDDLE $PERSON_NAME_LAST"

    fun mockAktorId(fnr: String): String {
        return "$MOCK_AKTORID_PREFIX$fnr"
    }
}
